kolibri/core/assets/src/views/breadcrumbs/index.vue

```diff
@@ -117,56 +137,53 @@
             const updatedCrumb = crumb;
             updatedCrumb.ref = crumbRefs[index];
             updatedCrumb.sensor = new ResizeSensor(updatedCrumb.ref, () => {
-              this.throttleUpdateCrumbs();
+              this.updateCrumbs();
```

Comment:
There might still be value in calling the throttled version. However, you could potentially lower the throttle interval to e.g. 100ms